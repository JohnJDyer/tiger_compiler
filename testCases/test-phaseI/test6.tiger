let
in
  printi(1 + 2 + 3 * 4);
end
