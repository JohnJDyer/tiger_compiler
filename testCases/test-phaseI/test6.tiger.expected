main:
	add, 1, 2, $t0
	mult, 3, 4, $t1
	add, $t0, $t1, $t2
	call, printi, $t2
	return, , , 
